pinc/ProjectTransition.inc

```diff
@@ -870,16 +871,14 @@ function round_release_collateral( $old_project, $project, $transition, $who )
 
 function bad_project_collateral( $old_project, $project, $transition, $who )
 {
-    global $site_abbreviation;
-    $body_blurb =
-"This project has been shut down.
-This is due to 10 or more problem reports, from at least
-3 unique users, noting errors or problems with this project.
-Please visit the Project Manager page to view a list
-of your bad projects and make any necessary changes.
-You will then be able to put the project back up on the site.";
-    maybe_mail_project_manager(
-        get_object_vars($project), $body_blurb, "$site_abbreviation: Bad Project");
+    configure_gettext_for_user($project->username);
+    $body_blurb_messages[] = _("This project has been shut down.");
+    $body_blurb_messages[] = _("This is due to 10 or more problem reports, from at least 3 unique users, noting errors or problems with this project.");
+    $body_blurb_messages[] = _("Please visit the Project Manager page to view a list of your bad projects and make any necessary changes.");
+    $body_blurb_messages[] = _("You will then be able to put the project back up on the site.");
```

Comment:
Ditto merging this into the `_()` above it so they are in the same paragraph.

Reply:
Looks like we were reviewing the same strings at the same time. You beat me to it. I think that this looks more like two paras, though. The first two together and the second two together.